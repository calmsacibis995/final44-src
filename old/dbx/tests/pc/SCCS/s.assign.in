h15233
s 00009/00000/00000
d D 5.1 90/06/25 15:36:29 bostic 1 0
c date and time created 90/06/25 15:36:29 by bostic
e
u
U
t
T
I 1
stop at 6
run
print a
assign a[1] = 'x'
print a
assign a = "xyzzy"
print a
cont
print a
E 1
