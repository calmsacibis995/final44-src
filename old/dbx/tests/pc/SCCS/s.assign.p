h16812
s 00007/00000/00000
d D 5.1 90/06/25 15:36:33 bostic 1 0
c date and time created 90/06/25 15:36:33 by bostic
e
u
U
t
T
I 1
program assign (input, output);

var a : array [1..100] of char;
begin
    a := 'blah';
    write(output, a);
end.
E 1
