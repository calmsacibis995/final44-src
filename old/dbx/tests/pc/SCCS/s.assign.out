h28111
s 00011/00000/00000
d D 5.1 90/06/25 15:36:31 bostic 1 0
c date and time created 90/06/25 15:36:31 by bostic
e
u
U
t
T
I 1
reading symbolic information ...
[1] stop at 6
[1] stopped in program at line 6
    6       write(output, a);
'blah                                                                                                ' 
'xlah                                                                                                ' 
'xyzzy' 
xyzzyexecution completed
'xyzzy' 

E 1
